src/api/app/models/project.rb

```diff
@@ -100,6 +100,7 @@ def autocomplete(search)
   scope :maintenance_release, -> { where("kind = 'maintenance_release'") }
   scope :home, -> { where("name like 'home:%'") }
   scope :not_home, -> { where.not("name like 'home:%'") }
+  scope :excluded, -> { where.not("name rlike ?", ::Configuration.excluded_projects) if ::Configuration.excluded_projects.present? }
```

Comment:
JFYI: ``rlike`` does not use MySQL index. However, probably hard to rewrite to something more performant if we don't know what the OBS Admins will do with it ....
https://www.psce.com/blog/2012/03/31/mysql-queries-with-regexp/